src/site/content/en/blog/build-for-webusb/index.md

```diff
@@ -0,0 +1,1014 @@
+descriptor], which provides basic information about the device. For the browser
+to fetch more information about the device it uses the `bVendorCode` value to
+issue additional requests to the device. The only request currently specified is
+`GET_URL` which returns a [URL descriptor].
+These are similar to string descriptors however are designed to encode URLs in
+the fewest bytes. A URL descriptor for `"https://google.com"` would look like
+this:
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+    <th>Value</th>
+    <th>Field</th>
+    <th>Description</th>
+    </tr>
+    <tr>
+    <td colspan="3">URL descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x0D</code></td>
+    <td>bLength</td>
+    <td>Size of this descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x03</code></td>
+    <td>bDescriptorType</td>
+    <td>URL descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x01</code></td>
+    <td>bScheme</td>
+    <td>https://</td>
+    </tr>
+    <tr>
+    <td><code>"google.com"</code></td>
+    <td>URL</td>
+    <td>UTF-8 encoded URL content</td>
+    </tr>
+  </table>
+</div>
+
+When your device is first plugged in the browser reads the BOS descriptor by
+issuing this standard `GET_DESCRIPTOR` control transfer:
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+    <th>bmRequestType</th>
+    <th>bRequest</th>
+    <th>wValue</th>
+    <th>wIndex</th>
+    <th>wLength</th>
+    <th>Data (response)</th>
+    </tr>
+    <tr>
+    <td><code>0b10000000</code></td>
+    <td><code>0x06</code></td>
+    <td><code>0x0F00</code></td>
+    <td><code>0x0000</code></td>
+    <td>*</td>
+    <td>The BOS descriptor</td>
+    </tr>
+  </table>
+</div>
+
+This request is usually made twice, the first time with a large enough `wLength`
+so that the host finds out the value of the `wTotalLength` field without
+committing to a large transfer and then again now that the full descriptor
+length is known.
+
+If the WebUSB Platform Capability descriptor has the `iLandingPage` field set to
+a non-zero value the browser then performs a WebUSB-specific `GET_URL` request
+by issuing a control transfer with the `bRequest` set to the `bVendorCode` value
+from the platform capability descriptor and `wValue` set to the `iLandingPage`
+value. The request code for `GET_URL` (`0x02`) goes in `wIndex`:
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+    <th>bmRequestType</th>
+    <th>bRequest</th>
+    <th>wValue</th>
+    <th>wIndex</th>
+    <th>wLength</th>
+    <th>Data (response)</th>
+    </tr>
+    <tr>
+    <td><code>0b11000000</code></td>
+    <td><code>0x01</code></td>
+    <td><code>0x0001</code></td>
+    <td><code>0x0002</code></td>
+    <td>*</td>
+    <td>The URL descriptor</td>
+    </tr>
+  </table>
+</div>
+
+Again, this request may be issued twice in order to first probe for the length
+of the descriptor being read.
+
+{% Aside %}
+Support for displaying a notification when a USB device is plugged in is not
+available yet on Android.
```

Comment:
@reillyeon Do you confirm Windows is now supported in Chrome 87 (stable)?

Reply:
Yes.